back/sssscs/src/main/java/com/ib/certificate/exception/InvalidCertificateTypeException.java

```diff
@@ -0,0 +1,12 @@
+package com.ib.certificate.exception;
+
+import com.ib.util.exception.EntityException;
+
+public class InvalidCertificateTypeException extends EntityException {
+	private static final long serialVersionUID = 3544584791580877663L;
```

Comment:
IntelliJ complains about this not being annotated with `@Serial`

Reply:
Done in [8fa7564](https://github.com/magley/sssscs/pull/3/commits/8fa75648ac249c21bfa128309d9fa1943762a56c).